product/reports/670_Performance by Asset Type - Middleware Servers/160_Web Metrics - hourly over last day.yaml

```diff
@@ -0,0 +1,61 @@
+---
+where_clause:
+dims:
+created_on: 2016-10-09 14:13:30.822035 Z
+reserved:
+title: "Web Metrics - hourly over the last day"
```

Comment:
is the title specific enough? it is not clear solely from it that it's regarding web metrics

Reply:
I don't have the big picture of whole system yet. Maybe I would add here that this metrics are from EAP: `EAP Web Metrics - XXX` maybe? Any tip?